src/vorta/views/archive_tab.py

```diff
@@ -665,16 +753,23 @@ def umount_result(self, result):
 
             if archive_name:
                 # unmount single archive
+                if archive_name in self.quick_mount_points:
+                    shutil.rmtree(self.quick_mount_points[archive_name])
+                    del self.quick_mount_points[archive_name]
                 del self.mount_points[archive_name]
                 row = self.row_of_archive(archive_name)
                 item = QTableWidgetItem('')
                 self.archiveTable.setItem(row, 3, item)
 
                 # update button
+                self.bmountarchive_update_menu()
                 self.bmountarchive_refresh()
             else:
                 # unmount repo
                 self.repo_mount_point = None
+                if self.quick_mount:
```

Comment:
So this variable is only used to decide if the folder get deleted? Why not use the name for that? What if I quick mount from one profile and then normal mount from another? Will it remove that folder too?